Cargo.toml

```diff
@@ -9,29 +9,28 @@ repository = "https://github.com/spruceid/oidc4vp-rs/"
 documentation = "https://docs.rs/oidc4vp/"
 
 [dependencies]
-jsonpath-rust = "0.2.0"
-serde = { version = "1.0.147", features = ["derive"] }
-serde_json = "1.0.87"
-ssi = { version = "0.6.0", default-features = false }
-thiserror = "1.0.37"
-lazy_static = "1.4.0"
-url = "2.3.1"
-rand = "0.8.5"
-isomdl = { git = "ssh://git@github.com/spruceid/isomdl.git", rev = "b2324b7" }
-#isomdl = {path = "../isomdl"}
-time = "0.3.21"
-p256 = "0.13.2"
-base64 = "0.13.1"
-x509-cert = "0.2.3"
-serde_cbor = "0.11.2"
-anyhow = "1.0.71"
-async-trait = "0.1.72"
-reqwest = "0.11.18"
+anyhow = "1.0.75"
+async-trait = "0.1.73"
+base64 = "0.21.4"
+did-web = "0.2.2"
+didkit = "0.6.0"
```

Comment:
I don't think libraries should use didkit, the APIs should instead accept a `DIDResolver` and let the downstream consumer choose what DIDs to support